server/src/main/java/org/opensearch/search/aggregations/bucket/FastFilterRewriteHelper.java

```diff
@@ -63,7 +63,7 @@ private FastFilterRewriteHelper() {}
 
     private static final Logger logger = LogManager.getLogger(FastFilterRewriteHelper.class);
 
-    private static final int MAX_NUM_FILTER_BUCKETS = 1024;
+    private static final int MAX_NUM_FILTER_BUCKETS = 24;
```

Comment:
I am wondering if we should add setting for number of buckets as well? Might not be widely used, but will be good to not needing code/jar change for testing the right number?